src/browser/transport.js

```diff
@@ -71,7 +71,23 @@ function _makeZoneRequest(accessToken, url, method, data, callback, requestFacto
   }
 }
 
+/* global RollbarProxy */
+function _proxyRequest(json, callback) {
+  var rollbarProxy = new RollbarProxy();
```

Comment:
I understand that the idea behind `RollbarProxy` is to come from other JS sources, just like in the Cordova SDK. Is that correct?

Reply:
Right. `RollbarProxy` is defined externally.